versions.yml

```
@@ -210,39 +209,39 @@ modGroups:
       desc: Adds a bunch of new particles
       url: https://cdn.modrinth.com/data/rI0hvYcd/versions/uhvQD6Ny/visuality-0.7.1%2B1.20.jar
 
-    - name: NotEnoughAnimations
+    - name: Not Enough Animations
```

```suggestion
    - name: NotEnoughAnimations
```

You can't just change the name.., it'll make launchy duplicate the mod, as the name is treated as the key/id for the mod.